NEWS.md

```diff
@@ -1,90 +1,83 @@
-3.20.1
+3.21.1
 ======
 
 ## Features
 
- * Add persist-tool (#2511)
- * Collectd destination (#1433)
- * Config reload status feedback (#2367)
- * Netskope parser (#2541)
- * Websense parser (#2471)
- * Json list support (#2536)
- * Xml-parser: add list support (#2544)
+ * Rdkafka destination (#2496)
+ * Added elasticsearch-http() destination (#2509)
```

Comment:
Added a native, http() destination based driver for elasticsearch, as an alternative of the Java one. Eventually, we expect this to replace the Java implementation.